src/axom/sidre/docs/sphinx/data_vs_metadata.rst

```diff
@@ -9,31 +9,33 @@
 Data vs. Metadata Concepts
 ===========================
 
-This section describes examples that illustrate some common Sidre usage 
-patterns involving buffers, groups, and views. In particular, it shows
+This section includes examples that illustrate some common Sidre 
+usage patterns involving buffers, groups, and views. In particular, it shows
 how data associated with a view is related to a buffer, how the lifetimes
 of data associated with a view and the buffer owning the data are related,
```

Comment:
Perhaps rephrase to 
```suggestion
of a view, its associated data, and the buffer owning the data are related,
```
or something like that.

Reply:
Because it's a little less repetitious and a little more concise, and also the view's lifetime matters here too.

Reply:
Thanks. Change made.